static/amp-video-iframe-videojs.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
```

Comment:
Does it make sense to preload 

- https://cdn.ampproject.org/video-iframe-integration-v0.js
- https://vjs.zencdn.net/7.3.0/video.js

(just to recommend some best practices)

Reply:
@aghassemi WDYT?